src/julia/options.py

```diff
@@ -134,9 +140,11 @@ class JuliaOptions(object):
     compile = Choices("compile", yes_no_etc("all", "min"))
     depwarn = Choices("depwarn", yes_no_etc("error"))
     warn_overwrite = Choices("warn_overwrite", yes_no_etc())
-    optimize = Choices("optimize", dict(zip(range(4), range(4))))
+    min_optlevel = Choices("min_optlevel", dict(zip(range(4), map(str, range(4)))))
+    optimize = Choices("optimize", dict(zip(range(4), map(str, range(4)))))
     inline = Choices("inline", yes_no_etc())
     check_bounds = Choices("check_bounds", yes_no_etc())
+    threads = Choices("threads", dict(zip(range(12), map(str, range(12)))))
```

Comment:
Needs to include "auto" as an option.

Also 12 threads max is a bit arbitrary. Is there no way to allow an arbitrary integer passed here?